### PR TITLE
Updates to SO contract

### PR DESCRIPTION
**Pull request by brittcyr**
Fix for lot size
Comment fixes
Remove unneeded checks that compute bumps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ Staking Options is a library for use in the Dual Finance ecosystem. One of the c
 
 ## How to integrate for projects
 
- 1. config
-	 - Project configures all the different parameters for the SO (Staking Option). This includes how long the options have until expiration, how many options are available, and other parameters needed for staking options.
- 2. initStrike
-	 - Project decides what strike to configure for options. A project can customize strikes to unlock taregted value for there community.
+1. config
+	 - Project configures all the different parameters for the SO (Staking Option). This includes how long the options have until expiration, how many options are available, lot size, and other parameters needed for staking options.
```
Comment (DonDuala) [inline README.md:8]: Should we mention determining call vs put ?

**Review by DonDuala — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brittcyr) [reply README.md:8]: There is no difference between a call and a put. You just flip the base and quote

Comment (brittcyr) [reply README.md:8]: Updated the readme

**Comment by brittcyr** [comment]
Fixes #5

**Pull request merged**